Chapter7/enum.erl

```diff
@@ -0,0 +1,37 @@
+-module(enum).
+-export([day/1, month/1]).
+
+% Exercise 7-8: Enumerated Types
+
+-define(DAY(Day),
+	case Day of
```

Comment:
it's no clear what exercise means by enumerated types.
maybe to bind names to values? something like that:
```erlang
-define(MONDAY, 1).
-define(TUESDAY, 2).
% ...
```